run

```diff
@@ -202,6 +229,10 @@ case $run_command in
     docker_django test
   ;;
   "clean")
+    running_containers="$(docker ps -aq --filter name=${project})"
```

Comment:
This isn't just running containers, the `-a` means it'll catch all stopped containers too (which is a good thing). We should probably call the variable `project_containers` or something.